gammapy/estimators/flux_point.py

```diff
@@ -159,40 +159,6 @@ def write(self, filename, sed_type="likelihood", **kwargs):
         table = self.to_table(sed_type=sed_type)
         table.write(filename, **kwargs)
 
-    @classmethod
-    def from_stack(cls, flux_points):
-        """Create flux points by stacking list of flux points.
-
-        The first `FluxPoints` object in the list is taken as a reference to infer
-        column names and units for the stacked object.
-
-        Parameters
-        ----------
-        flux_points : list of `FluxPoints`
-            List of flux points to stack.
-
-        Returns
-        -------
-        flux_points : `FluxPoints`
-            Flux points without upper limit points.
-        """
-        reference = flux_points[0].to_table(sed_type="dnde")
-
-        tables = []
-
-        for fp in flux_points:
-            table = fp.to_table(sed_type="dnde")
-            for colname in reference.colnames:
-                column = reference[colname]
-                if column.unit:
-                    table[colname] = table[colname].quantity.to(column.unit)
-            tables.append(table[reference.colnames])
-
-        table_stacked = vstack(tables)
-        table_stacked.meta["SED_TYPE"] = "dnde"
-        table_stacked.sort("e_ref")
-        return cls.from_table(table=table_stacked, sed_type="dnde")
-
     @staticmethod
     def _convert_loglike_columns(table):
         # TODO: check sign and factor 2 here
```

Comment:
From the documentation of GADF (https://gamma-astro-data-formats.readthedocs.io/en/latest/spectra/flux_points/index.html#likelihood-columns) the loglike format does not include the 2 ("Log-Likelihood value of the best-fit model.", +log and not -log! So it should be negative).
In `to_table' of the `flux_point' code, the profile is made with the cash statistics (to be checked once again). And thus, the stored data are -2 * log... (see https://cxc.cfa.harvard.edu/sherpa/statistics/#cash). The stored values should be positive
So, I think that we should add the sign minus here.
In order to be sure about the input format (their could not follow the definition of GADF), one could test if the input data are really negative!

PS: I did not find any web documentation for `FluxMaps' and `CountsStatistic'